.codespell.skip

```diff
@@ -8,4 +8,5 @@
 *.ico
```

Comment:
probably need to update the CI job to add this too?

is there a way we can use this file as input to the job, that would make it easier to keep up to date

Reply:
can't see anything from a quick glance at the action README

Reply:
updated the job. I don't see a way other than moving back to using the makefile for everything (which I'd personally prefer). This way there's no chance of skewing the jobs.